biz.aQute.junit/src/aQute/junit/plugin/ProjectTesterImpl.java

```diff
@@ -57,7 +62,9 @@ public boolean prepare() throws Exception {
 			Collection<String> testnames = getTests();
 			if (testnames.size() > 0) {
 				launcher.getRunProperties()
-					.put(TESTER_NAMES, Processor.join(testnames));
+					.put(TESTER_NAMES, Processor.join(testnames.stream()
```

Comment:
It seems busy to stream/map the list into a list and then join into a string. We can go straight there

```java
.put(TESTER_NAMES, testnames.stream()
  .map(ProjectTesterImpl::maybeQuote)
  .collect(Collectors.joining(",")));
```

Reply:
:+1: wasn't sure if `Processor.join()` was doing some other magic.